ocrd/decorators.py

```diff
@@ -49,8 +49,8 @@ def cli(mets_url):
         click.option('-p', '--parameter', type=click.Path()),
         click.option('-J', '--dump-json', help="Dump tool description as JSON and exit", is_flag=True, default=False),
         click.option('-l', '--log-level', help="Log level",
-            type=click.Choice(['OFF', 'ERROR', 'WARN', 'INFO', 'DEBUG', 'TRACE']),
-            default='INFO', callback=_set_root_logger_version),
+                     type=click.Choice(['OFF', 'ERROR', 'WARN', 'INFO', 'DEBUG', 'TRACE']),
+                     default='INFO', callback=_set_root_logger_version),
```

Comment:
I believe this must not have a `default` any more. If it does have a default, then the callback will **always** be triggered, overriding all configuration by the `INFO` level. (I have tested this, too.)

Reply:
Well spotted, thanks. We should document that the default log level `INFO` and explicitly providing `--log-level INFO` cause different behaviors when using core.

Reply:
should be fixed and tested now.

Reply:
It is, thanks. But do we really need that bitter pill of not having a default (in the user sense, not in the `click.option` implementation sense) any more? We would have to set the respective level to the root logger in a configuration file for that now. Doesn't the spec say that INFO should be the default setting? 

(I think all we need is an additional `logging.getLogger('').setLevel(logging.INFO)` in ocrd/logging.py.)

Reply:
ok, I guess you did then :-) I am on your 8b99b512 ... there is only a setting for `PIL` in there.

Reply:
Since we already stumbled across that other twist, how about adding `self.assertEqual(logging.getLogger('').getEffectiveLevel(), logging.INFO)` as first assertion to `test_loglevel_override`?

Reply:
That should also be in the Code. We think alike apparently :D Ill push tomorrow sorry for the confusion thanks for reviewing!

Reply:
Works like a charm, thanks! 

No idea what causes these new assertion failures. Tests pass for me with python 3.6.5, 3.4.3, 2.7.15rc1, and 2.7.6.

Reply:
Thanks for testing. I guess travis has a different test harness. It works on CircleCI and Scrutinizer, but travis possibly hooks into `setUp` and sets root logger level to `DEBUG` or something like that.

Reply:
Fixed with explicitly initiating loggin in `setUp`.